README.md

```diff
@@ -0,0 +1,51 @@
+# L3: The Lightweight Logging Library
```

Comment:
Rewrote in `md` format, with minor inline edits and rewrites for better readability.